components/AddToHomescreen.tsx

```diff
@@ -13,14 +13,14 @@ export default function AddToHomescreen() {
     }
   }
   return !closeBanner ? (
-    <div className="fixed sm:hidden bottom-0 inset-x-0 pb-2 sm:pb-5">
-      <div className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8">
+    <div className="fixed inset-x-0 bottom-0 pb-2 sm:hidden sm:pb-5">
+      <div className="px-2 mx-auto max-w-7xl sm:px-6 lg:px-8">
         <div className="p-2 rounded-lg shadow-lg sm:p-3" style={{ background: "#2F333D" }}>
-          <div className="flex items-center justify-between flex-wrap">
-            <div className="w-0 flex-1 flex items-center">
-              <span className="flex p-2 rounded-lg bg-opacity-30 bg-brand">
+          <div className="flex flex-wrap items-center justify-between">
+            <div className="flex items-center flex-1 w-0">
+              <span className="flex p-2 rounded-lg bg-opacity-30 bg-brand text-brandtext">
```

Comment:
we could rename this to text-brand right?

Reply:
Agreed `text-brandtext` sounds really weird IMO.